hoomd/md/NeighborListGPUBinned.h

```diff
@@ -76,6 +76,19 @@ class PYBIND11_EXPORT NeighborListGPUBinned : public NeighborListGPU
             return m_cl->getSortCellList();
             }
 
+        #ifdef ENABLE_MPI
+
+        virtual void setCommunicator(std::shared_ptr<Communicator> comm)
+            {
+            // call base class method
+            NeighborList::setCommunicator(comm);
+
+            // set the communicator on the internal cell lists
+            m_cl->setCommunicator(comm);
+            }
+
+        #endif
+
```

Comment:
This seems to be the same implementation as `NeighborListBinned`, so we should remove it.

Reply:
`NeighborListGPUBinned` is not a subclass of `NeighborListBinned`, so `setCommunicator` is not inherited from it.
https://github.com/glotzerlab/hoomd-blue/blob/e8eeb42ecd11d5f36b8e2221c0c0ee79e34385d8/hoomd/md/NeighborListGPUBinned.h#L31